src/api/roles/client/telemetry.py

```
@@ -140,17 +145,347 @@ def query_workouts(
+    # can share the same telemetry row so we don't dedupe on telemetry —
+    # each completed-workout entry counts toward the burned total.
+    workout_rows = db.exec(
+        select(CompletedWorkout, ClientTelemetry, CompletedWorkoutActivity)
+        .join(ClientTelemetry, CompletedWorkout.client_telemetry_id == ClientTelemetry.id)
+        .join(CompletedWorkoutActivity, CompletedWorkoutActivity.id == CompletedWorkout.completed_workout_details_id, isouter=True)
+        .where(ClientTelemetry.client_id == acc.client_id)
+    ).all()
+
+    burned = 0.0
+    workout_count = 0
+    for completed, telemetry, details in workout_rows:
```

Ruff (0.15.12)

[warning] Loop control variable `completed` not used within loop body

Rename unused `completed` to `_completed`

(B007)